.prettierrc.json

```diff
@@ -11,7 +11,8 @@
           "./scripts/prettier-swagger-plugin"
         ],
         "parser": "json-swagger",
-        "printWidth": 20
+        "printWidth": 20,
+        "endOfLine": "auto"
```

Comment:
What does endOfLine do?

Reply:
It's to specify line ending: https://prettier.io/docs/en/options.html#end-of-line

From Prettier 2.0.0, the default value changed from `auto` to `lf`. To avoid necessary changes, set it to `auto` explicitly.